extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -303,18 +307,20 @@ export class RefreshManager {
    * seconds to milliseconds.
    *
    * @param {(number|string)} refreshInterval
+   * @param {boolean=} useMsecs Uses ms as unit of time rather than seconds.
    * @return {?number}
    */
-  checkAndSanitizeRefreshInterval_(refreshInterval) {
+  checkAndSanitizeRefreshInterval_(refreshInterval, useMsecs) {
     const refreshIntervalNum = Number(refreshInterval);
-    if (isNaN(refreshIntervalNum) ||
-        refreshIntervalNum < MIN_REFRESH_INTERVAL) {
+    const upperBound = useMsecs ?
```

Comment:
I am still nervous about the interchange between millis and seconds here.  Let's change this so that the variables/const referenced are ALWAYS millis so that the real change you need to make is that getPublisherSpecifiedRefreshInterval_ should return in millis by converting the seconds value given in the pub config to millis.